sdk/servicebus/Azure.Messaging.ServiceBus/src/Amqp/AmqpReceiver.cs

```diff
@@ -1366,8 +1367,8 @@ private void OnReceiverLinkClosed(object receiver, EventArgs e)
         public override async Task OpenLinkAsync(CancellationToken cancellationToken)
         {
             _ = await _retryPolicy.RunOperation(
-                static async (receiveLink, timeout, _) =>
-                    await receiveLink.GetOrCreateAsync(timeout).ConfigureAwait(false),
+                async (receiveLink, timeout, _) =>
+                    await receiveLink.GetOrCreateAsync(timeout, cancellationToken).ConfigureAwait(false),
```

Comment:
Cobbled together without IDE support so be warned ;) 
```suggestion
                async (receiveLink, timeout, token) =>
                    await receiveLink.GetOrCreateAsync(timeout, token).ConfigureAwait(false),
```